src/services/Api.tsx

```diff
@@ -0,0 +1,57 @@
+import axios from 'axios';
```

Comment:
It should be .ts file as long as there's no JSX syntax inside. I think there's not here.